tests/test_polydata.py

```diff
@@ -154,13 +154,11 @@ def test_invalid_init():
         pyvista.PolyData(np.array([1]))
 
     with pytest.raises(TypeError):
-        pyvista.PolyData(np.array([1]), 'woa')
+        pyvista.PolyData([1, 2, 3], 'woa')
 
-    with pytest.raises(TypeError):
+    with pytest.raises(ValueError):
         pyvista.PolyData('woa', 'woa')
 
-    with pytest.raises(TypeError):
-        pyvista.PolyData('woa', 'woa', 'woa')
 
```

Comment:
Boost coverage a bit:
```suggestion

    poly = pyvista.PolyData()
    with pytest.raises(ValueError):
        pyvista.PolyData(poly, 'woa')

    with pytest.raises(TypeError):
        pyvista.PolyData({'woa'})
```

Reply:
Was just working on that.  I'll commit this and add some more.

Reply:
Sorry, I got an off-by-one-line error in the suggestion :( One of the lines should be below, not above.